### DISCUSSION TITLE
I know this may be a known to most, but how do I change the system prompt from the config.lua file ?

### DISCUSSION THREAD

**Discussion by Marvinus** [yetone/avante.nvim#755] [Q&A]
I am using LazyVim and installed avante.nvim. But it seems like Claude is responding in french every time. Så I need to change the system prompt to ensure that it respond in English. If I change it directly in the config.lua file then I cannot update avante. So where would I change the prompt in relation to LazyVim ?

**Comment by aarnphm** [comment]
Check avanterules for specific planning or editing rules.

if you want to change system prompt check wiki

edit: link for custom system prompt https://github.com/yetone/avante.nvim#custom-prompts
